developer-model-service/docs/index.md

```diff
@@ -1,5 +1,5 @@
 # Table of Contents
 - [Supported Models](./models.md)
-- [Access & Usage Information](./code/access.md)
+- [Access & Usage Information](./access.md)
```

Comment:
Since we moved access from `/code` the link in the sidebar is broken, I don't believe you changed it in this PR so I can't reference it but it would be this: https://github.com/redhat-ai-dev/model-catalog/blob/main/multi-model-service/mkdocs.yml#L6

Reply:
Yeah, forgot to fix that, oops